server/utils/dateUtils.ts

```diff
@@ -25,10 +25,14 @@ export class DateFormats {
    * @param date JS Date object.
    * @returns the date in the to be shown in the UI: "Thursday, 20 December 2012".
    */
-  static dateObjtoUIDate(date: Date, options: { format: 'short' | 'long' } = { format: 'long' }) {
+  static dateObjtoUIDate(date: Date, options: { format: 'short' | 'medium' | 'long' } = { format: 'long' }) {
     if (options.format === 'long') {
       return format(date, 'cccc d MMMM y')
-    } else {
+    } 
+    if (options.format === 'medium') {
+      return format(date, 'd MMMM y')
```

Comment:
maybe more of a content question but do we always want the full month name here, rather than an abbreviated name ('MMM')? Just wondering how the longer ones will end up looking in the table